bahamas/app/gateway.py

```diff
@@ -0,0 +1,80 @@
+from bahamas.app import app
+from flask import request, Response
+
+from bahamas.app import db
+from bahamas.app.models import Client, Invoice
+from bahamas.app.validations import *
+
+NEEDED_ARGS = ['email', 'fiscal_id', 'name']
+
+
+@app.route('/store-bahamas-client/<invoice_id>')
+def register(invoice_id):
+    for key in NEEDED_ARGS:
+        if key not in request.args:
+            return "Missing or invalid argument. Please check if all parameters are correct", 404
```

Comment:
Hey! I chose the 404 status because that was what I thought it would be most accurate "Not found all arguments that are needed"
The request it's just a GET request, maybe I should have specified it on the route

Reply:
@djguu hey.

404 NOT FOUND
The origin server did not find a current representation for the target resource or is not willing to disclose that one exists.

It means that the request was processed but could not find the resource. 

Which HTTP error code would you choose if you want to let the client know that some parameters were missing in the request?
With a 404 I could assume that the endpoint does not exists.

Reply:
hey @RicCastelhano ! I was thinking and the best error code I can send it's the 400 Bad Request since it can be used for invalid parameters values in the request.